src/components/Text.js

```diff
@@ -166,3 +166,34 @@ export const BoldText = styled(RegularText)`
       font-family: bold-italic;
     `};
 `;
+
+export const HeadlineText = styled(RegularText)`
+  font-weight: 700;
+  font-family: bold;
+
+  ${(props) =>
+    props.medium &&
+    css`
+      font-size: ${normalize(18)};
+      line-height: ${normalize(23)};
+    `};
```

Comment:
i think this is not needed. there are two cases i can see. medium or small. medium is something we do not have in our code base yet.
i suggest using the style of medium as general for `HeadlineText` and small as variation, where necessary.

Reply:
Yes. I realised this better in our update last week and I'm going to fix it.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/920/commits/24147097e638fbb4faab803fab473fc7bbe10503